pom.xml

```diff
@@ -796,6 +796,17 @@ under the License.</licenseText>
             </excludes>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-pmd-plugin</artifactId>
```

Comment:
Is that related ?

Reply:
Yes, older PMD ctaps out on Java 24

Reply:
nice to see things going in same direction
- https://github.com/apache/maven/pull/2331